roles/rsyslog/roles/input_roles/ovirt/tasks/main.yaml

```diff
@@ -1,17 +1,16 @@
 ---
-- name: Set oVirt facts
-  set_fact:
-    rsyslog_role_packages: "{{ rsyslog_ovirt_prereq_packages|flatten|d([]) }}+{{ rsyslog_ovirt_packages|flatten|d([]) }}"
-    rsyslog_role_rules: "{{ rsyslog_ovirt_rules }}"
-
+# Deploy configuration files
 - name: Install/Update role packages and generate role configuration files to role subdir
+  vars:
+    __rsyslog_packages: "{{ __rsyslog_ovirt_prereq_packages | d([]) + __rsyslog_ovirt_packages | d([]) }}"
```

Comment:
@nhosoi Why did you remove the "flatten"?

Reply:
Please see the comment https://github.com/linux-system-roles/logging/pull/87#discussion_r384707842